test/parallel/test-fs-mkdir-recursive-eaccess.js

```diff
@@ -26,8 +26,7 @@ function makeDirectoryReadOnly(dir) {
   let accessErrorCode = 'EACCES';
   if (common.isWindows) {
     accessErrorCode = 'EPERM';
-    execSync(`icacls ${dir} /inheritance:r`);
-    execSync(`icacls ${dir} /deny "everyone":W`);
+    execSync(`icacls ${dir} /deny "everyone:(OI)(CI)(DE,DC,AD,WD)"`);
```

Comment:
```suggestion
    execSync(`attrib +r ${dir}`);
```

Reply:
This doesn't work locally for me (doesn't throw).

Reply:
Does it also remove the file successfully?

Reply:
Yes, I also tried with /s and /d

Reply:
It's fine, I don't mind experimenting and simplifying things to see if they work out.

Reply:
Also, `icacls` doesn't stop me from deleting the directory: https://github.com/RaisinTen/tests/runs/2602063819?check_suite_focus=true

Reply:
Specifically, the test doesn't try  to remove the folder - one test tries to create a subfolder and one tries to remove a file inside of the directory.

Reply:
I think that DE might be wrong, as it's not listed here:
https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/icacls
It _is_ listed in older docs that I found.